source/common/runtime/runtime_features.cc

```diff
@@ -110,6 +110,8 @@ constexpr const char* disabled_runtime_features[] = {
     "envoy.reloadable_features.remove_legacy_json",
     // Sentinel and test flag.
     "envoy.reloadable_features.test_feature_false",
+    // Reject virtual host-specific configuration when filter doesn't support it.
+    "envoy.reloadable_features.check_unsupported_typed_per_filter_config",
```

Comment:
My understanding is that runtime features should default to true.  They provide a way for devs affected by a breaking change to temporarily rollback the behavior.  Once we have given time to find and address all the breakages, we'll remove the runtime feature checks and only keep the new behavior.

Out of curiosity, are there any existing tests that fail when this runtime feature is enabled?

Reply:
> My understanding is that runtime features should default to true. They provide a way for devs affected by a breaking change to temporarily rollback the behavior. Once we have given time to find and address all the breakages, we'll remove the runtime feature checks and only keep the new behavior.


As `https://github.com/envoyproxy/envoy/issues/15043#issuecomment-778703531` said, I thought we should set ti as false default, then it won't break existing user. But I may miss understanding that.


> Out of curiosity, are there any existing tests that fail when this runtime feature is enabled?

I tested with the default value is true. In my env, only get one test failed 'IoSocketHandleImplIntegration.LastRoundTripIntegrationTest', I don't think it is related.

Reply:
cc @htuch

Having the runtime flag so affected devs can disable the functionality while they fixed their config should be enough.  I think that the default should be true.

And I agree that the 'IoSocketHandleImplIntegration.LastRoundTripIntegrationTest' failure is not related to this change.  That's a non-hermetic test that does network communication.  I think I'll file an issue about it.

Reply:
I think default true is fine, as long as we have the runtime for now.

Reply:
cc @htuch

Having the runtime flag so affected devs can disable the functionality while they fixed their config should be enough.  I think that the default should be true.

And I agree that the 'IoSocketHandleImplIntegration.LastRoundTripIntegrationTest' failure is not related to this change.  That's a non-hermetic test that does network communication.  I think I'll file an issue about it.

Reply:
let me change the default value as true, thanks.